pkg/kn/config/config.go

```diff
@@ -97,7 +97,7 @@ var globalConfig = config{}
 // GlobalConfig is the global configuration available for every sub-command
 var GlobalConfig Config = &globalConfig
 
-// bootstrapConfig reads in config file and boostrap options if set.
+// bootstrapConfig reads in config file and bootstrap options if set.
```

Comment:
`bootstrapConfig` in the comment line should be `BootstrapConfig`, beginning with a capital letter.